static/app/components/replays/utils.tsx

```diff
@@ -24,14 +24,14 @@ const HOUR = 60 * MINUTE;
  * @returns Unix timestamp of the adjusted timestamp, in milliseconds
  */
 export function relativeTimeInMs(timestamp: moment.MomentInput, diffMs: number): number {
-  return moment(timestamp).diff(moment.unix(diffMs)).valueOf();
```

Comment:
this was messed up.

Asking for `@param diffMs Number of milliseconds` and then putting it into `moment.unix()` 

https://momentjs.com/docs/#/parsing/unix-timestamp/
> To create a moment from a Unix timestamp (seconds since the Unix Epoch), use moment.unix(Number).